CMakeLists.txt

```diff
@@ -3,12 +3,14 @@ project(warehouse_ros)
 
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/" ${CMAKE_MODULE_PATH})
 
-find_package(catkin REQUIRED
+find_package(catkin REQUIRED COMPONENTS
   geometry_msgs
   roscpp
-  rostest
   rostime
   std_msgs)
+if (CATKIN_ENABLE_TESTING)
+  find_package(catkin REQUIRED COMPONENTS rostest)
```

Comment:
This will overwrite the variable starting with `catkin_` which is likely not working since the packages found above won't be part of `catkin_INCLUDE_DIRS` and `catkin_LIBRARIES` anymore.

Instead use `find_package(rostest REQUIRED)`.